io.itsterminal.WebPConverter.json

```diff
@@ -0,0 +1,34 @@
+{
+  "app-id": "io.itsterminal.WebPConverter",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "webp_converter.py",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--device=dri"
+  ],
+  "modules": [
+    {
+      "name": "webp_converter",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -D webp_converter.py /app/bin/webp_converter.py",
+        "install -D data/icons/io.itsterminal.WebPConverter.svg /app/share/icons/hicolor/scalable/apps/io.itsterminal.WebPConverter.svg",
+        "install -D desktop/io.itsterminal.WebPConverter.desktop /app/share/applications/io.itsterminal.WebPConverter.desktop",
+        "install -D data/appstream/io.itsterminal.WebPConverter.appdata.xml /app/share/metainfo/io.itsterminal.WebPConverter.metainfo.xml"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/MichaelScuteri/webp-converter-gnome.git",
+          "tag": "v1.1.0",    
```

Comment:
tag does not exist

Reply:
yep sorry, i didn't push the tag. It has been pushed now.